### ISSUE TITLE
Ensures aria-hidden elements do not contain focusable elements from reCaptcha

### ISSUE THREAD

**Issue by langyanbin0314** [google/recaptcha#442]
Title: WCAG 1.3.1,WCAG 4.1.2: Ensures aria-hidden elements do not contain focusable elements (.rc-anchor-normal-footer)
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, aria-hidden-focus

Issue: Ensures aria-hidden elements do not contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

Element path: iframe[width="\32 56"];.rc-anchor-normal-footer

Snippet: <div class="rc-anchor-normal-footer" aria-hidden="true">

How to fix: 
Fix all of the following:
  Focusable content should have tabindex='-1' or be removed from the DOM

Environment: Chrome version 88.0.4324.150

====

This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by rowan-m** [comment]
Repo is just for PHP client issues. This is about the reCAPTCHA service.
